cfme/tests/cli/test_evmserverd.py

```diff
@@ -30,26 +30,28 @@ def test_evmserverd_stop(ssh_client):
         * For 5.5+: Really call ``service evmserverd status`` and check that the mentions of
             stopping the service are present.
     """
-    server_names = {server["Server Name"] for server in ssh_client.status["servers"]}
-    assert ssh_client.run_command("service evmserverd stop").rc == 0
+    server_names = {server["Server Name"] for server in appliance.ssh_client.status["servers"]}
```

Comment:
This is raising a KeyError upstream for 'Server Name' but that's outside your intended scope here.